x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/KerberosRealmTests.java

```diff
@@ -110,7 +110,6 @@ public void testLookupUser() {
         assertThat(future.actionGet(), is(nullValue()));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/32768")
     public void testKerberosRealmWithInvalidKeytabPathConfigurations() throws IOException {
         final String keytabPathCase = randomFrom("keytabPathAsDirectory", "keytabFileDoesNotExist", "keytabPathWithNoReadPermissions");
```

Comment:
I don't understand why we are using randomization here as opposed to simply running every case each time. Randomization is good for finding edge cases where we randomize some input data. I don't think it's good for poor man's test coverage to ensure all branches are covered, we really should be exercising those on every test run.